### ISSUE TITLE
Add support for tweet-provider

### ISSUE THREAD

**Issue by alazymeme** [pajbot/pajbot2-discord#29]
https://github.com/pajbot/tweet-provider

Some people using pajbot1 also use tweet-provider, so it could probably also help with the discord bot also

**Comment by alazymeme** [comment]
This needs to be done first, then rate limits could be an issue: https://github.com/pajbot/pajbot2-discord/issues/30

**Comment by alazymeme** [comment]
Merged into a single issue
